src/initial_conditions/initial_conditions.jl

```diff
@@ -351,10 +376,80 @@ function (initial_condition::RisingThermalBubbleProfile)(params)
     return local_state
 end
 
+overwrite_initial_conditions!(initial_condition::InitialCondition, args...) =
+    (return nothing)
+
+"""
+    overwrite_initial_conditions!(initial_condition, Y, thermo_params, config)
```

Comment:
cc @Sbozzolo

Reply:
Isn't this function recompuing some hydrostatic balance? This should be mentioned and explained somewhere, otherwise how could people understand what's the point of these lines

```julia
    ᶜ∂lnp∂z = @. -thermo_params.grav /
       (TD.gas_constant_air(thermo_params, TD.PhasePartition(ᶜq_tot)) * ᶜT)
    ᶠlnp_over_psfc = zeros(face_space)
    Operators.column_integral_indefinite!(ᶠlnp_over_psfc, ᶜ∂lnp∂z)
    ᶠp = p_sfc .* exp.(ᶠlnp_over_psfc)
```
There is no mention anywhere (code, comments, GitHub) of the challanges you were facing and why this code looks like this instead of just reading the pressure.

Reply:
Fixed.